orchestra/models.py

```diff
@@ -28,16 +76,92 @@ class Certification(models.Model):
             A longer description of the certification.
         required_certifications ([orchestra.models.Certification]):
             Prerequisite certifications for possessing this one.
+        workflow (orchestra.models.Workflow):
+            The workflow the certification is scoped to.
     """
-    slug = models.CharField(max_length=200, unique=True)
+    slug = models.CharField(max_length=200)
     name = models.CharField(max_length=200)
     description = models.TextField()
-    required_certifications = models.ManyToManyField('self',
-                                                     blank=True)
+    required_certifications = models.ManyToManyField(
+        'self',
+        blank=True,
+        related_name='dependent_certifications',
+        symmetrical=False)
+    workflow = models.ForeignKey(Workflow, related_name='certifications')
 
     def __str__(self):
         return '{}'.format(self.slug)
 
+    class Meta:
+        app_label = 'orchestra'
+        unique_together = (('workflow', 'slug'),)
+
+
+class WorkflowStep(models.Model):
```

Comment:
My preference is for being explicit--if I had my way, Task would be renamed :smile: 

But I don't feel strongly about it if you want to change it.